README.md

```diff
@@ -1,11 +1,51 @@
-# migaloo-chain
+# Migaloo
```

Comment:
nit: would love to see a logo here eventually

Reply:
A banner is coming don't you worry!